wannierberri/__integrate.py

```diff
@@ -130,15 +138,20 @@
         additional_parameters[calc][key] = val[0]
         additional_parameters_description[calc][key] = val[1]
 
-key='kpart'
-for calc in calculators_trans:
-    if calc.endswith('_ocean'):
-        additional_parameters[calc][key] = 500
-        additional_parameters_description[calc][key] = (
-             'Separate k-points of the FFT grid into portions ' + 
-             '(analog of ksep in the system class, but acts in different calculators)'  +
-             'decreasing this parameter helps to save memory in some cases' +
-                'while performance is usually unafected' )
+#for calc in calculators_trans:
```

Comment:
Please Delete these comments if they are not relevant any more. If not, please add some comment on why there are commented out but still there.

Reply:
Done, removed many commented lines in the code